capsule/src/CapsuleWallet.ts

```diff
@@ -352,15 +352,16 @@ export abstract class CapsuleBaseWallet {
    * We initialize the manager late to ensure the userID is available.
    * @category Private
    */
-  private async initSessionManagerIfNeeded() {
+  private async initSessionManagerIfNeeded(webauth = false) {
     if (!this.sessionManager) {
       const userId = await this.getUserId();
       if (!userId) {
         throw Error('UserId not available during initializing session key');
       }
       this.sessionManager = new SessionManager(
         userId,
-        this.getChallengeStorage(userId)
+        this.getChallengeStorage(userId),
+        webauth
```

Comment:
```suggestion
        webauth,
```

See Google style guide 5.2.1 Use trailing commas